<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="startTaskBtn">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>开始学习</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopTaskBtn">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>10</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>结束学习</string>
   </property>
  </widget>
  <widget class="QComboBox" name="chooseTask">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="taskEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addTaskBtn">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>添加任务</string>
   </property>
  </widget>
  <widget class="QPushButton" name="showTableBtn">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>10</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>显示学习情况</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>681</width>
     <height>401</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>671</width>
       <height>38</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>你的酒馆别对我打烊 (非商业使用)</family>
       <pointsize>24</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="text">
      <string>好好学习，天天向上</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="timeLabel">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>551</width>
       <height>281</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>87</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>661</width>
       <height>331</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteBtn">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>0</y>
       <width>91</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>删除记录</string>
     </property>
    </widget>
    <widget class="QPushButton" name="amendBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>91</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>提交修改</string>
     </property>
    </widget>
    <widget class="QPushButton" name="amendCancelBtn">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>0</y>
       <width>91</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>撤销修改</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2"/>
  </widget>
  <widget class="QPushButton" name="deleteTaskBtn">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>删除任务</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
